### PR TITLE
redirect package submissions to Factory

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

File: package/libstorage-ng.changes [hunk 1]
```diff
@@ -1,17 +1,4275 @@
--------------------------------------------------------------------
```
Comment (jreidinger) [inline package/libstorage-ng.changes]: is such big change to changes required? it looks quite strange for me.

Comment (wfeldt) [inline package/libstorage-ng.changes]: We didn't write any changes file so far.

Comment (jreidinger) [inline package/libstorage-ng.changes]: I see there is some old entries in changes. If there is agreement on it, then I am fine with it.

Comment (wfeldt) [inline package/libstorage-ng.changes]: Yes, that has to be sorted out in obs in the first submission.

**Comment by shundhammer** [comment]
LGTM

**Pull request merged**